pocketmine.json

```diff
@@ -0,0 +1,36 @@
+{
+    "PocketMine": {
+        "containers": {
+            "pocketmine": {
+                "image": "magicalyak/pocketmine",
+                "launch_order": 1,
+                "opts": [
+		  [
+			"-it",
+			"--restart=always",
```

Comment:
Thank you @magicalyak . I have two questions here. 

1. Why ```-it```?

2. The rock-on framework automatically adds ```--restart=unless-stopped```, not sure if you've noticed it. Given that, any reason, why we'd want ```--restart=always```?